error in patch to other-dates/$id

```sh
app-1           | 2026/02/16 20:07:08 [ERROR] [WEB] ↳ Response: 500 Internal Server Error for PATCH /api/v1/other-dates/4
kindredcard-db  | 2026-02-16 20:07:08.259 CST [42] ERROR:  could not determine data type of parameter $3
kindredcard-db  | 2026-02-16 20:07:08.259 CST [42] STATEMENT:  
kindredcard-db  |                       UPDATE other_dates
kindredcard-db  |                       SET event_name = $1, event_date = $2
kindredcard-db  |                       WHERE id = $4
```

--- a/internal/db/dates.go
+++ b/internal/db/dates.go
@@ -98,7 +98,7 @@ func (d *Database) UpdateContactOtherDate(userID int, body models.ContactDateJSO
 		UPDATE other_dates
 		SET %s
 		WHERE id = $%d
-	`, strings.Join(updates, ", "), argIndex+1)
+	`, strings.Join(updates, ", "), argIndex)
 
 	_, err := d.db.Exec(query, args...)
 	if err != nil {

